yarn.lock

```diff
@@ -3728,14 +3728,9 @@
     "@types/jest" "*"
 
 "@types/tough-cookie@*":
-  version "4.1.3"
-  resolved "https://registry.yarnpkg.com/tough-cookie/-/tough-cookie-4.1.3.tgz#97b9adb0728b42280aa3d814b6b999b2ff0318bf"
-  integrity sha512-aX/y5pVRkfRnfmuX+OdbSdXvPe6ieKX/G2s7e98f4poJHnqH3281gDPm/metm6E/WRamfx7WC4HUqkWHfQHprw==
-  dependencies:
-    psl "^1.1.33"
-    punycode "^2.1.1"
-    universalify "^0.2.0"
-    url-parse "^1.5.3"
+  version "4.0.5"
```

Comment:
Is this correct? Looks like a downgrade.

Reply:
I raised this as well: https://github.com/gravitational/teleport/pull/36792#discussion_r1455887514

The prior version was seemingly in error, once we merge I will review what Dependabot says